scripts/scripts.js

```diff
@@ -30,13 +30,48 @@ function buildHeroBlock(main) {
   }
 }
 
+/**
+* Builds floating image block and prepends to main in a new section.
+* @param {Element} main The container element
+*/
+function buildFloatingImages(main) {
+  main.querySelectorAll('.section-metadata').forEach((metadata) => {
+    let style;
```

Comment:
Looks like this could have a scoping issue.  I don't see this `style` being assigned a value, and the `other style` inside the `every` function call is in a closure.  This might always be uninitialized right now.